lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/JWTVerifier.java

```diff
@@ -74,7 +74,7 @@ private void verifyJWTClaimsSet(
                                 .issuer(clientAuthenticationConfig.get("issuer"))
                                 .audience(clientAuthenticationConfig.get("audience"))
                                 .build(),
-                        new HashSet<>(Arrays.asList("exp", "nbf", "sub")));
```

Comment:
we don't send nbf in the token req

Reply:
It would be best to add an overload to the public verifyJWT with a requiredClaims HashSet parameter.
Then nbf or anything else can be passed in on a per invocation basis

Reply:
We'll handle `nbf` as a separate task

Reply:
we should check the nbf for the Auth JAR